From 89d4a38dde99a6b141e90860fca594a9ac66336b Mon Sep 17 00:00:00 2001
From: Jonas Gorski <jogo@openwrt.org>
Date: Tue, 16 Jul 2013 14:02:57 +0200
Subject: [PATCH V2 05/13] MIPS: BMIPS: select CPU_SUPPORTS_HIGHMEM

All BMIPS CPUs support HIGHMEM, so it should be selected by CPU_BMIPS.

Signed-off-by: Jonas Gorski <jogo@openwrt.org>
---
 arch/mips/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/mips/Kconfig
+++ b/arch/mips/Kconfig
@@ -1466,6 +1466,7 @@ config CPU_BMIPS
 	select IRQ_CPU
 	select SWAP_IO_SPACE
 	select WEAK_ORDERING
+	select CPU_SUPPORTS_HIGHMEM
 	help
 	  Support for BMIPS3300/4350/4380 and BMIPS5000 processors.
 
@@ -1563,7 +1564,6 @@ config CPU_BMIPS4380
 
 config CPU_BMIPS5000
 	bool
-	select CPU_SUPPORTS_HIGHMEM
 	select MIPS_CPU_SCACHE
 	select SYS_SUPPORTS_SMP
 	select SYS_SUPPORTS_HOTPLUG_CPU
